src/uniprotkb/components/protein-data-views/__tests__/FreeTextView.spec.tsx

```diff
@@ -1,12 +1,23 @@
-import { render } from '@testing-library/react';
+import { screen } from '@testing-library/react';
+import customRender from '../../../../shared/__test-helpers__/customRender';
 
 import FreeTextView from '../FreeTextView';
 
-import FreeTextUIData from './__mocks__/freeTextUIData';
+import freeTextUIData from './__mocks__/freeTextUIData';
 
 describe('FreeText component', () => {
   test('should render free text CC', () => {
-    const { asFragment } = render(<FreeTextView comments={FreeTextUIData} />);
+    const { asFragment } = customRender(
+      <FreeTextView comments={freeTextUIData} />
+    );
     expect(asFragment()).toMatchSnapshot();
+
+    // PubMed links
+    expect(screen.getAllByRole('link')).toHaveLength(2);
```

Comment:
I'm also wondering whether it would be worth testing edge cases like comment with no PubMed id to prevent accidental regression.

Reply:
I'm assuming you wanted more combinations of the test, rather than splitting this specific assertion into 3, right?
I ended up making 3 tests, with similar assertions, but different props (plain text+evidence tags / text with inline pubmeds+evidence tags / plain text+no evidence tag).

Reply:
With Jest I always find it weird to have multiple assertions in the same test - I know it's different from unit tests, and I guess as long as you're testing a small workflow it's ok. In this case though, I think 2 of the assertions were already covered by the snapshot?

Reply:
Ah, I see, I'll split them into multiple suites then, for each combination of prop, and then test the things in different tests.
I'd be keen on removing the snapshots here, as I'm already asserting on the important bit which are the links and evidence tags, what do you think?